### PR TITLE
fix: bad segment edit link

### PR DESCRIPTION
**Pull request by gastonfournier**
## About the changes
Fixes a bad link which leads to editing a global segment from project settings mimicking https://github.com/Unleash/unleash/blob/67928209eff0c00c22919007b860de5c3d0119e9/frontend/src/component/segments/EditSegmentButton/EditSegmentButton.tsx#L22-L28

### CODE REVIEW

**Review by daveleek — APPROVED** [review]
Great! LGTM!

File: frontend/src/component/common/PermissionButton/PermissionButton.tsx [hunk 1]
```diff
@@ -55,7 +55,16 @@ const ProjectEnvironmentPermissionButton: React.FC<IProjectPermissionButtonProps
...
         );
 
-        return <BasePermissionButton {...props} access={access} ref={ref} />;
+        return (
+            <BasePermissionButton
+                {...props}
+                access={access}
+                ref={ref}
+                sx={{
+                    backgroundColor: 'red',
```
Comment (nunogois) [inline frontend/src/component/common/PermissionButton/PermissionButton.tsx:64]: I assume that this is only present for debugging reasons and we don't want this to be merged.

File: frontend/src/component/common/PermissionButton/PermissionButton.tsx [hunk 2]
```diff
@@ -132,7 +141,16 @@ const PermissionButton: React.FC<IPermissionButtonProps> = React.forwardRef(
...
             );
         }
-        return <RootPermissionButton {...props} ref={ref} />;
+        return (
+            <RootPermissionButton
+                {...props}
+                ref={ref}
+                sx={{
+                    backgroundColor: 'blue',
+                    textDecoration: 'overline',
```
Comment (nunogois) [inline frontend/src/component/common/PermissionButton/PermissionButton.tsx:150]: Same here.

File: frontend/src/component/common/PermissionButton/PermissionButton.tsx (hunk 1 above)
Comment (gastonfournier) [reply frontend/src/component/common/PermissionButton/PermissionButton.tsx:64]: thanks! This one slipped :red_square:

**Review by nunogois — APPROVED** [review]
Great 👍

**Pull request merged**